README.md

```diff
@@ -81,12 +83,25 @@ MEMORY
 }
 ```
 
+For non-secure mode with TFM (Trusted Firmware-M):
+
+```ld
+MEMORY
+{
+  /* Trusted Firmware-M (TF-M) is flashed at the start */
+  FLASH         : ORIGIN = 0x00008000, LENGTH = 0xf8000
+  RAM     (rwx) : ORIGIN = 0x2000C568, LENGTH = 0x33a98
```

Comment:
Usually we don't specify the length with hex. This is not wrong per se, but let's keep the readme more conventional.